test/integration/CommentBlockEnd/comment block end.js

```diff
@@ -0,0 +1,2 @@
+*/
+]
```

Comment:
Notice how some of these files have the little red symbol? It means there's no newline at the end of the file. Could you add one?